config/install/oe_corporate_blocks.ec_data.footer.yml

```diff
@@ -17,9 +17,6 @@ service_navigation:
     href: 'https://commission.europa.eu/legal-notice/vulnerability-disclosure-policy_en'
     label: 'Report an IT vulnerability'
 legal_navigation:
-  -
```

Comment:
This is not good because it removes the link from horizontal links where it is supposed to be (check reqs page [here](https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?spaceKey=EWPP&title=New+visual+identity+2025+-+EC+Core+Footer) of the new EC core footer)